pkgs/top-level/all-packages.nix

```diff
@@ -18582,6 +18578,10 @@ in
 
   appeditor = callPackage ../applications/misc/appeditor { };
 
+  apostrophe = callPackage ../applications/editors/apostrophe {
```

Comment:
Maybe this should get an entry in `pkgs/top-level/aliases.nix`?

Reply:
I'm not sure. `uberwriter` never entered a NixOS main release, so it only breaks for users `nixos-unstable`. I'm not sure if it is common practice to add an alias in this situation.